pkg/neonvm/controllers/metrics.go

```diff
@@ -50,7 +51,7 @@ func MakeReconcilerMetrics() ReconcilerMetrics {
 				Name: "reconcile_failing_objects",
 				Help: "Number of objects that are failing to reconcile for each specific controller",
 			},
-			[]string{"controller", OutcomeLabel},
+			[]string{"controller", OutcomeLabel, "retried"},
```

Comment:
IMO "retried" is confusing here -- if I'm looking at the metrics, and I see that there's failing objects with `retried="true"` / `retried="false"`, I would assume that's relating to whether this is the first time the object has failed.

Any ideas for alternative names? maybe `stuck_without_retry`?